xcollection/main.py

```diff
@@ -79,3 +80,31 @@ def values(self) -> typing.Iterable[xr.Dataset]:
 
     def items(self) -> typing.Iterable[typing.Tuple[str, xr.Dataset]]:
         return self.datasets.items()
+
+    def choose_all(self, data_vars: typing.Union[str, typing.List[str]]) -> 'Collection':
```

Comment:
Shall we do

```
def choose(self, data_vars, *, mode="all")
```

and `mode` could be `"any"`

Reply:
Should the default be `mode="any"` and then the user needs to explicitly specify when they expect the variable to be in every dataset?

Reply:
> Should the default be `mode="any"` and then the user needs to explicitly specify when they expect the variable to be in every dataset?


Done!